### PR TITLE
Add options in singing karaoke

### PR DESCRIPTION
**Pull request by dbhoite**
Please review this, @casey-collab

### CODE REVIEW

File: english/karaoke/karaoke/sing.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-you are not very good at singing!
\ No newline at end of file
+you are not very good at singing!
+
+You continue to sing anyway. The drunk audience breaks into a standing ovation
+on your courage.  Your eyes meet with the girl of your dreams.
+
+Do you:
+[propose to her](propose-to-her/propose-to-her.md)
+[puke on the mike](puke-on-the-mike/puke-on-the-mike.md)
```
Comment (casey-collab) [inline english/karaoke/karaoke/sing.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by dbhoite** [comment]
Fixed newlines. Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**